src/plotly_recipes.jl

```diff
@@ -72,8 +72,15 @@ function _dataframe_plots_internal(
         if bar
             plot_kwargs[:marker_color] = seriescolor[ix]
         end
+        data_to_plot = plot_data[:, ix]
+        if sum(data_to_plot) >= 0
+            sign_group = 0
+        else
+            sign_group = 10
+        end
+
         if stack
-            plot_kwargs[:stackgroup] = string(plot_length + 1)
+            plot_kwargs[:stackgroup] = string(plot_length + 1 + sign_group)
```

Comment:
Not sure I fully understand this line. I get that we want the negative and non-negative data to go in different groups and that adding `sign_group` achieves that, but why are we naming the groups based on `plot_length`, and why the magic number `sign_group = 10`?

Reply:
Adding a 10 is such that is still compatible with the other uses of plot data frames.